locales/en.default.schema.json

```diff
@@ -2745,6 +2745,9 @@
     "quick-order-form": {
       "name": "Quick order form",
       "settings": {
+        "enable_sticky_header": {
```

Comment:
When items are added to cart, there is some jumping: https://screenshot.click/12-57-nfqnp-9n0sg.mp4